pkg/client/cli/cmds_intercept.go

```diff
@@ -522,6 +537,20 @@ func (is *interceptState) EnsureState(ctx context.Context) (acquired bool, err e
 		}
 	}()
 
+	if is.args.previewEnabled && is.args.cmdLineIngress && is.args.previewSpec.Ingress == nil {
+		ingress := &manager.IngressInfo{}
+		ingress.Host = is.args.ingressHost
+		ingress.Port = is.args.ingressPort
```

Comment:
Some sanity checks would be welcome here. An ingress host must be a valid hostname (see `var hostRx` declared further down). The port cannot be zero.